src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+      ]
+    },
+    {
+      "what": [
+        "a configuration option on charts to disable or enable animation."
+      ],
+      "why": [
+        "to allow users more chart configuration options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/806/commits/f9cc910e5a769538c83fa96acb53f04af0f9efe8"
+      ]
+    },
+    {
+      "what": [
+        "BETWEEN and NOT BETWEEN operator support for single value fields(number and date-time record fields)."
+      ],
+      "why": [
+        "to extend record filtering support for range filtering of numeric and date-time for single value field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/813/commits/2027409b0d8875a48fd0017264ba03f5535733ec"
+      ]
+    },
+    {
+      "what": [
+        "new support for a fully customizable navigation page block with multiple navigation item type and customizable options for the navigation list items."
+      ],
+      "why": [
+        "to extend the possibilities of offered blocks to the users and to have the ability to show one or two level of dynamic and static navigation."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/41f1283e9d393393c651b8ac5a28cae09733f1fd"
+      ]
+    },
+    {
+      "what": [
+        "warning message for unsaved changes before navigating to another page."
+      ],
+      "why": [
+        "to prevent losing unsaved changes by unintentionally navigating to another page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/780/commits/a5d66312d4b6cc6505d883a5e3eebbc638a9ae71"
+      ]
+    },
+    {
+      "what": [
+        "ability to drill down chart, by clicking on a data entry inside on the chart.",
+        "Based on the configuration it can either filter an existing record list on the page, that is showing the records from the same module as the chart, or open a new record list in a modal."
+      ],
+      "why": [
+        "to make it easier for users to filter record lists based on chart data, in order to view the details of said data."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/877/commits/cfc4ab7424ed1a131bb34cfaed8c4ca18eef1b07"
+      ]
+    },
+    {
+      "what": [
+        "support for conditional fields.",
+        "Fields in records can now be displayed based on evaluated expressions."
+      ],
+      "why": [
+        "to give users more control on how records are displayed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/770/commits/1d286aff83f012e33cabc3456ffa175f7c105eb7"
+      ]
+    },
+    {
+      "what": [
+        "support to allow users to navigate to previous and next record page.",
+        "Added a checkbox to enable the record page pagination on the configurator record list."
```

Comment:
Consider explaining this a bit more; sounds very vague and would have me question the feature if I wasn't involved with it.